thirdparty/proto/v1/ipns/ipns_entry.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package sonrio.sonr.thirdparty.ipns;
+
+option go_package = "github.com/sonr-io/sonr/thirdparty/proto/v1/ipns";
```

Comment:
`thirdparty` should be `third_party` in the project structure if we're trying to stay conventional.